### DISCUSSION TITLE
download cx_freeze

### DISCUSSION THREAD

**Discussion by abekomi** [marcelotduarte/cx_Freeze#1881] [Q&A]
i want to download cx_freeze onto a drive to put on a computer that does not have internet connection i cant figure it out ,how do you do it?

**Comment by marcelotduarte** [comment]
https://pypi.org/project/cx-Freeze/#files

**Reply by abekomi** [reply]
but where do you put all the files that you download which path do you put in?

**Reply by DiddiLeija** [reply]
You could use pip, almost like online packages. Once you downloaded the files into the new pc, open your terminal and run `python -m pip [cx_Freeze file]` :)

**Reply by abekomi** [reply]
that would work? i thought pip is like only for online packages?

**Reply by marcelotduarte** [reply]
Please read the documentation. Yes, it works! pip install packages. 
At https://pip.pypa.io/en/stable/cli/pip_install/ see examples # 8.
https://cx-freeze.readthedocs.io/en/latest/installation.html
